server/controller/src/main/resources/sql/schema.sql

```diff
@@ -36,4 +36,56 @@ CREATE TABLE IF NOT EXISTS project_info (
     PRIMARY KEY (id),
     INDEX idx_project_name(project_name) USING BTREE,
     INDEX idx_create_user_id(owner_id) USING BTREE
+);
+
+CREATE TABLE IF NOT EXISTS agent_info (
+    id bigint NOT NULL AUTO_INCREMENT COMMENT '主键，自增',
```

Comment:
comment use `English Word`?

Reply:
ok. done.

Reply:
OK  The chinese comment has bean removed. The column name  is enough.